backend/fms_core/template_importer/importers/sample_qc.py

```diff
@@ -0,0 +1,82 @@
+from fms_core.models import Process, Protocol, PropertyType
+
+from ._generic import GenericImporter
+from fms_core.template_importer.row_handlers.sample_update import SampleRowHandler
+
+from .._utils import float_to_decimal_and_none
+
+PROPERTIES_STARTING_INDEX = 7
+
+class SampleQCImporter(GenericImporter):
+    SHEETS_INFO = [
+        {
+            'name': 'SampleQC',
+            'headers': ['Sample Container Barcode', 'Sample Container Coord', 'Initial Volume (uL)',
+                        'Measured Volume (uL)', 'Volume Used (uL)', 'Concentration (ng/uL)', 'NA Quantity (ng)',
+                        'RIN (for RNA only)', 'Electrophoresis Instrument', 'Quality Flag', 'Quantitation Instrument',
+                        'Quantity Flag', 'QC Date', 'Comment']
+        },
+    ]
+
+    def __init__(self):
+        super().__init__()
+        self.initialize_data_for_template()
+        self.properties_starting_index = PROPERTIES_STARTING_INDEX
+
+    def initialize_data_for_template(self):
+        self.preloaded_data = {'process': None, 'process_properties': {}}
+
+        self.preloaded_data['process'] = Process.objects.create(protocol=Protocol.objects.get(name="Sample Quality Control"),
+                                                                comment="Samples' Quality Control (imported from template)")
+
+        # Preload PropertyType objects for the sample qc in a dictionary for faster access
+        try:
+            protocol = Protocol.objects.get(name="Sample Quality Control")
```

Comment:
Would make more sense to get the protocol first then use it to set the preloaded_data['process']...